src/locales/en.json

```diff
@@ -172,5 +172,16 @@
   "setting.push.posts" : "all posts",
   "settings.push.thisDevice": "This device ({name})",
   "settings.title": "Settings",
+  "team.modal.editTitle.title": "Team Name",
+  "team.modal.button.save": "Save",
+  "team.modal.editInfo.title": "Team Info",
+  "team.modal.editCustomStyles.title": "Custom styles",
+  "team.modal.editCustomStyles.help": "Here you can override the global css style variables of the theme. See [the Ionic Theming Guide](https://ionicframework.com/docs/theming/css-variables) and the [handy Color Generator](https://ionicframework.com/docs/theming/color-generator).",
```

Comment:
Does markdown work in i18n? I suspect not even html is working here...

I think we have to opt for [`<i18n-t..>` with slots](https://vue-i18n-next.intlify.dev/guide/advanced/component.html#slots-syntax-usage) for this....